src/dlite-errors.h

```diff
@@ -8,14 +8,14 @@ typedef enum {
   dliteUnknownError = -1,
   dliteIOError = -2,
   dliteRuntimeError = -3, /* originally -1 */
-  dliteIndexError = -4,
-  dliteTypeError = -5,
+  dliteIndexError = -4, // e.g. x = [1,2,3], x[7]
```

Comment:
We should have a good description of each error code. The examples are good, but we should use C syntax.
We should also use the `/*!< ... */` doxygen inline comment style, such that these comments will be included in the doxygen-generated documentation.

```suggestion
  dliteIndexError = -4,      /*!< Index out of range. Ex: int x[3] = {1,2,3}; x[7]; */
```

Reply:
Latest push has this implemented (I think)